src/snippets/s5.html

```diff
@@ -214,7 +214,7 @@ <h4>Material</h4>
 </ul>
 <div style="left: 0px; width: 100%; height: 0px; position: relative; padding-bottom: 56.25%; max-width: 800px;" data-ephox-embed-iri="https://youtu.be/jqFMWNk8HR4"><iframe style="border: 0; top: 0; left: 0; width: 100%; height: 100%; position: absolute;" src="https://www.youtube.com/embed/jqFMWNk8HR4?rel=0" scrolling="no" allowfullscreen="allowfullscreen"></iframe></div>
 <p>&nbsp;</p>
-<div style="max-width: 800px;" data-ephox-embed-iri="https://google.com/google.jpg"><img src="https://www.google.com/logos/google.jpg" /></div>
+<div style="max-width: 800px;"><img src="https://www.google.com/logos/google.jpg" /></div>
```

Comment:
whats the reason for this change

Reply:
is this a bug?

Reply:
Not sure if that is expected for mediaembedded elements.